### ISSUE TITLE
gradle.properties is getting updated and Versions.kt stops updating when running buildSrcVersions

### ISSUE THREAD

**Issue by digitalsanctum** [jmfayard/buildSrcVersions#92]
**Checklist**

- [x] I verified that the task `./gradlew :dependencyUpdates` works correctly. If that's not the case, it's better to submit a bug-report to https://github.com/ben-manes/gradle-versions-plugin 
- [ ] I tried to reproduce the bug within `sample-groovy`, `sample-kotlin` or `sample-versionsOnlyMode` 
- [x] I have run with `$ ./gradlew --scan buildSrcVersions` or `$./gradlew --scan refreshVersions` to [provide a Build Scan URL](https://dev.to/jmfayard/the-one-gradle-trick-that-supersedes-all-the-others-5bpg)
 
**Describe the bug**

I've just upgraded to `0.6.0` and ran `./gradlew buildSrcVersions`. It produced an updated Versions.kt as expected. I updated Versions.kt with newer versions and updated Gradle from 5.5.1 to 5.6.2. I ran `./gradlew buildSrcVersions` again and it looks like Versions.kt did not update again. 
Note the console output for `dependencyUpdates` is correct.

The gradle.properties file also keeps getting updated every time I run `./gradlew buildSrcVersions`. My understanding is this should only happen when running `./gradlew refreshVersions`?

**To Reproduce**

Build scan URL: https://scans.gradle.com/s/lvzdafpbzky7y/console-log

see above

**Expected behavior**
I expected `Versions.kt` to get updated after I manually updated versions. 

I expected Gradle `runningVersion` in `Versions.kt` to get updated from 5.5.1 to 5.6.2 after I upgraded gradle wrapper via `gradle wrapper --gradle-version 5.6.2`. 

**Additional context**

**Comment by jmfayard** [comment]
Thanks for the report, I will look into it next week

Le sam. 28 sept. 2019 à 17:46, Shane Witbeck <notifications@github.com> a
écrit :

**Comment by jmfayard** [comment]
Hello @digitalsanctum  I can't reproduce what you describe. 

I tried this:

```bash
git clone git@github.com:jmfayard/buildSrcVersions.git
cd buildSrcVersions/sample-groovy/
./gradlew buildSrcVersions 
# gradle.properties is not modified
./gradlew refreshVersions
# gradle.properties is modified
```

**Comment by digitalsanctum** [comment]
Sorry. I forgot to mention that I ran `refreshVersions` at some point and I think the problem started after that. I just reproduced with the following steps:

```
git clone git@github.com:jmfayard/buildSrcVersions.git
cd buildSrcVersions/sample-groovy/
./gradlew buildSrcVersions 
# gradle.properties is not modified
./gradlew refreshVersions
# gradle.properties is modified
# update build.gradle and change guava version to 14.0 from 15.0
./gradlew buildSrcVersions
# Versions.kt did not update
```

**Comment by jmfayard** [comment]
@digitalsanctum I was surprised, but you were totally right. 
Is it supposed to be fixed in version 0.6.2. 
Can you confirm that this is the case?

**Comment by digitalsanctum** [comment]
@jmfayard Sorry to say I'm having a different issue now. I upgraded to 0.6.2 and ran `./gradlew buildSrcVersions`. This resulted in both my `Libs` and `Versions` files getting corrupted. All the constants got prepended with the string: `from-editorconfig-file`. Here's an example:

```from-editorconfig-fileconst val clikt: String = "com.github.ajalt:clikt:2.1.0"```

I also tried running `./gradlew refreshVersions && ./gradlew buildSrcVersions` which updated `gradle.properties` but still corrupted `Libs`
